cosmos/txbuilder.go

```diff
@@ -17,22 +17,30 @@ type TxBuilder struct {
 
 // NewTxBuilder returns a new initialized TxBuilder.
 func NewTxBuilder(accNumber, accSeq uint64, kb keys.Keybase, chainID string) TxBuilder {
+	// sdktypes.NewDecCoin("mesg", sdktypes.NewInt())
+	// sdktypes.NewDecCoinFromDec("mesg", sdktypes.NewDec())
+
 	return TxBuilder{
 		authtypes.NewTxBuilder(
 			authutils.GetTxEncoder(codec.Codec),
 			accNumber,
 			accSeq,
-			1000000,
+			flags.DefaultGasLimit,
 			flags.DefaultGasAdjustment,
 			true,
 			chainID,
 			"",
 			sdktypes.NewCoins(),
-			sdktypes.DecCoins{},
+			sdktypes.NewDecCoins(sdktypes.NewCoins(sdktypes.NewCoin("mesg", sdktypes.NewInt(1)))),
```

Comment:
Should be put in config

Reply:
This requires some refactoring, just to not add gasPrice as argument everywhere to already "argument-bolted" methods. Leaving decision to you.